tests/admin_widgets/tests.py

```diff
@@ -468,7 +468,7 @@ def test_readonly_fields(self):
 class ForeignKeyRawIdWidgetTest(TestCase):
 
     def test_render(self):
-        band = models.Band.objects.create(name='Linkin Park')
+        band = models.Band.objects.create(pk=1, name='Linkin Park')
```

Comment:
It's not a good practice to manually assign primary keys like this. It can cause problems with database sequences.